src/main/java/listeners/TestListener.java

```diff
@@ -0,0 +1,117 @@
+package listeners;
+
+import io.qameta.allure.Attachment;
+import org.apache.log4j.Logger;
+import org.openqa.selenium.OutputType;
+import org.openqa.selenium.TakesScreenshot;
+import org.testng.ITestContext;
+import org.testng.ITestListener;
+import org.testng.ITestResult;
+import webdriver.DriverManager;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+public class TestListener implements ITestListener {
+
+    private static final String LOGFILE_NAME = "logfile.log";
+    private static final Logger logger = Logger.getLogger("Test listener logger");
+    private final VideoRecorder recorder = new VideoRecorder();
+    private String testName;
```

Comment:
Is that possible to avoid name variable being global? AVOID changeable global variables